poly/src/domain/mod.rs

```diff
@@ -70,8 +81,11 @@ pub trait EvaluationDomain<F: FftField>:
     /// Return the group inverse of `self.group_gen()`.
     fn group_gen_inv(&self) -> F;
 
-    /// Return the inverse of the multiplicative generator of `F`.
-    fn generator_inv(&self) -> F;
+    /// Return the offset that defines this domain.
+    fn offset(&self) -> F;
```

Comment:
Would it be accurate to call this `coset_offset`, or `offset_from_group`? Otherwise the name is kind of mysterious